### ISSUE TITLE
Make visible the task notes

### ISSUE THREAD

**Issue by byrnereese** [mikedamage/hayfever-chrome#4]
I am not sure this is something you can fix, but I will give you the feedback nonetheless. I use the Mac OS X Widget for all of my timing needs. I then use a 3rd party app to force the widget to run on top of my desktop (rather than on my OS X dashboard). 

It provides a list of timers just as your extension does, however this is what I see: 

https://skitch.com/byrnereese/rthat/dock

What has always bothered me about this view is that the "task" in harvest is always very generic, e.g. "Development" or "Project Management". I take great care to input a more descriptive task note, and would much rather see that in my listing because it is more specific and often helpful to me. 

I was hoping perhaps that would be more evident in your chrome extension.

Does that make sense?

Generally speaking I find the text weight and emphasis in the table listing to not necessary be best for _me_. :) Right now you have:

Client Name (bold): Task name

I would prefer:

Task Note (slightly bigger font)
Client Name/Task (slightly smaller font)

In other words, I want to make the list quicker and easier to scan, and for me the most salient piece of information is captured in the note, not in the task name.
